docs/source/installation.rst

```diff
@@ -101,3 +101,14 @@ Windows
   - Run ``python -c "import gpkit.tests; gpkit.tests.run()"``
       - If attempting to run the tests results in ``ValueError: Unknown solver ''.`` and ``python -c "import gpkit"`` prints "Could not find settings file", then run ``python -c "import gpkit.build; gpkit.build.build_gpkit()"``, to look for and install solvers. After doing so (it should say which solvers have been found), run     ``python -c "import gpkit.tests; gpkit.tests.run()"`` again.
   - If you want units support, install pint with ``pip install pint``.
+
+
+Updating GPkit between releases
+===============================
+
+Active developers may wish to install the `latest GPkit <http://github.com/convexopt/gpkit>` directly from the source code on Github. To do so,
+
+  - Run ``pip uninstall gpkit`` to uninstall your existing GPkit.
+  - Run ``git clone https://github.com/convexopt/gpkit.git`` to clone the GPkit repository.
+  - Run ``pip install -e gpkit`` to reinstall GPkit.
+  - Run ``python -c "import gpkit.tests; gpkit.tests.run()"``

```

Comment:
no I like it! wilco.

While I'm doing that -- The other nitpick / question is: doesn't the user have to be in a specific dir (one level above gpkit) for the pip install to work? This is probably a non-issue for people who clone the repo as described above, but for someone who pulls to update an existing repo, it might be confusing?